[Bug]: bdemount missing from worker Dockerfile

### What steps will reproduce the bug?

Build latest docker from Dockerfile and startup with docker-compose. 

```
turbinia-worker      | 2023-11-06 15:40:55 [ERROR] There was a problem processing arguments: Job dependency bdemount not found in $PATH for the job partitionenumerationjob. Please install the dependency or disable the job.
```

Seems we are not installing bdemount anymore (used to be apt-get installed from libbde-tools.

### What is the expected behavior?

clean worker startup with bdemount dependency check passing

### What do you see instead?

```
turbinia-worker      | 2023-11-06 15:40:55 [ERROR] There was a problem processing arguments: Job dependency bdemount not found in $PATH for the job partitionenumerationjob. Please install the dependency or disable the job.
```

### Additional information

_No response_

--- a/docker/worker/Dockerfile
+++ b/docker/worker/Dockerfile
@@ -12,11 +12,8 @@ RUN apt-get update && apt-get -y upgrade && apt-get -y install \
     git \
     john \
     john-data \
-    libbde-utils \
-    libfsapfs-utils \
     libleveldb1d \
     libleveldb-dev \
-    libluksde-utils \
     libssl-dev \
     libterm-readline-gnu-perl \
     lvm2 \
@@ -36,10 +33,14 @@ RUN pip3 install poetry
 #   bulkextractor
 #   docker-explorer
 #   Sleuthkit
+#   bdemount, apfs and luks tools
 RUN add-apt-repository -y ppa:gift/$PPA_TRACK
 RUN apt-get update && apt-get -y install \
     bulk-extractor \
     docker-explorer-tools \
+    libbde-tools \
+    libfsapfs-tools \
+    libluksde-tools \
     sleuthkit \
     && apt-get clean && rm -rf /var/cache/apt/* /var/lib/apt/lists/*
 

--- a/docker/worker/Dockerfile
+++ b/docker/worker/Dockerfile
@@ -16,6 +16,7 @@ RUN apt-get update && apt-get -y upgrade && apt-get -y install \
     libfsapfs-utils \
     libleveldb1d \
     libleveldb-dev \
+    libluksde-utils \
     libssl-dev \
     libterm-readline-gnu-perl \
     lvm2 \

--- a/docker/worker/Dockerfile
+++ b/docker/worker/Dockerfile
@@ -12,6 +12,7 @@ RUN apt-get update && apt-get -y upgrade && apt-get -y install \
     git \
     john \
     john-data \
+    libbde-utils \
     libleveldb1d \
     libleveldb-dev \
     libssl-dev \

--- a/docker/worker/Dockerfile
+++ b/docker/worker/Dockerfile
@@ -13,6 +13,7 @@ RUN apt-get update && apt-get -y upgrade && apt-get -y install \
     john \
     john-data \
     libbde-utils \
+    libfsapfs-utils \
     libleveldb1d \
     libleveldb-dev \
     libssl-dev \

